test/FunctionalTests/MvcTests/SampleTestFixture.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.IO;
```

Comment:
Is this needed when `GENERATE_BASELINES` isn't set? Wondering about Remove and Sort Usings doing the Wrong Thing<sup>:tm:</sup>.